website/docs/configuration/modules.html.md

```diff
@@ -161,56 +227,83 @@ about how modules can be used, created, and published is included in
 In a configuration with multiple modules, there are some special considerations
 for how resources are associated with provider configurations.
 
-While in principle `provider` blocks can appear in any module, it is recommended
-that they be placed only in the _root_ module of a configuration, since this
-approach allows users to configure providers just once and re-use them across
-all descendent modules.
-
 Each resource in the configuration must be associated with one provider
-configuration, which may either be within the same module as the resource
-or be passed from the parent module. Providers can be passed down to descendent
-modules in two ways: either _implicitly_ through inheritance, or _explicitly_
-via the `providers` argument within a `module` block. These two options are
-discussed in more detail in the following sections.
-
-In all cases it is recommended to keep explicit provider configurations only in
-the root module and pass them (whether implicitly or explicitly) down to
-descendent modules. This avoids the provider configurations from being "lost"
-when descendent modules are removed from the configuration. It also allows
-the user of a configuration to determine which providers require credentials
-by inspecting only the root module.
+configuration. Provider configurations, unlike most other concepts in
+Terraform, are global to an entire Terraform configuration and can be shared
+across module boundaries. Terraform modules can be declared only in a root
```

Comment:
> Terraform modules can be declared only in a root Terraform module
I am not sure what you were going for here, maybe something else relating to passing providers to a module?  Either way, this statement is incorrect.

Reply:
Oh, yes! This was supposed to be "Terraform _providers_ can be declared only ..."! :upside_down_face: